examples/meta/src/regression/chaidtree.sg.in

```diff
@@ -0,0 +1,28 @@
+File f_feats_train    = read_csv("@SHOGUN_DATA@/regression_1d_linear_features_train.dat")
+File f_feats_test     = read_csv("@SHOGUN_DATA@/regression_1d_linear_features_test.dat")
+File f_labels_train   = read_csv("@SHOGUN_DATA@/regression_1d_linear_labels_train.dat")
+
+#![create_features]
+Features feats_train = create_features(f_feats_train)
+Features feats_test = create_features(f_feats_test)
+Labels labels_train = create_labels(f_labels_train)
+#![create_features]
+
+#![set_feature_types]
+IntVector ft(1)
+ft[0] = 2
+#![set_feature_types]
+
+#![create_machine]
+Machine chaidtree = create_machine("CHAIDTree", labels=labels_train, dependent_vartype=2, feature_types=ft, num_breakpoints=50)
```

Comment:
actually the error is here: if you have a look at the parameter registration they all have a `m_` prefix. You should remove that `m_` for the parameter name
https://github.com/shogun-toolbox/shogun/blob/7483101c9ec1f18b6dcee5af5fad099349f4a78a/src/shogun/multiclass/tree/CHAIDTree.cpp#L1395-L1404

Reply:
Oh okay, I'll do that.